integration/integrate-with-google-cloud-scc.adoc

```diff
@@ -0,0 +1,23 @@
+[id="integrate-with-google-cloud-scc"]
+= Integrating with Google Cloud Security Command Center
+include::modules/common-attributes.adoc[]
+:context: integrate-with-google-cloud-scc
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://cloud.google.com/security-command-center/[Google Cloud Security Command Center] (Cloud SCC), you can forward alerts from {product-title} to Cloud SCC.
+This guide explains how to integrate {product-title} with Cloud SCC.
+
+The following steps represent a high-level workflow for integrating {product-title} with Cloud SCC.
+
+. Register a new Security Source with Google Cloud.
+. Provide the Source ID and service account key to {product-title}.
```

Comment:
Same question on "Source ID" (and other places in the PR, if you do lowercase it)